graylog2-web-interface/src/views/components/widgets/Widget.jsx

```diff
@@ -49,11 +60,21 @@ type State = {
   configChanged?: boolean,
   editing: boolean,
   oldConfig?: WidgetConfig,
+  showCopyToDashboard: boolean,
 };
 
 class Widget extends React.Component<Props, State> {
+  static _visualizationForType(type) {
```

Comment:
This is unrelated to the PR, but while you are touching it: These can be moved out of the class and be just plain functions defined in the top scope of this file.